common/utils/src/main/resources/error/error-classes.json

```diff
@@ -370,6 +370,27 @@
     ],
     "sqlState" : "42710"
   },
+  "CREATE_VIEW_COLUMN_ARITY_MISMATCH" : {
```

Comment:
Could you align this new error class with the existing `INSERT_COLUMN_ARITY_MISMATCH` and output columns?

Reply:
Done.